cmd/internal/serverutil/main.go

```diff
@@ -69,8 +68,6 @@ type Main struct {
 	StatsPrefix string
```

Comment:
References to "REST" in the comment for HTTPEndpoint above should also possibly be removed.